Godeps/Godeps.json

```diff
@@ -0,0 +1,6 @@
+{
+	"ImportPath": "k8s.io/test-infra",
```

Comment:
We use both godeps and the vendor directory. It looks like this PR is correct in that.